<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFrameworkClass</class>
 <widget class="QMainWindow" name="MyFrameworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyFramework</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="vl_UI">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="top_wgt" native="true">
      <property name="minimumSize">
       <size>
        <width>1014</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="left_wgt" native="true">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>111</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QPushButton" name="pb_min_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>64</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>当前用户</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color:transparent;
	background-image: url(:/MyFramework/images/logo4.png);
    border:0px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="center_wgt_2" native="true">
         <property name="minimumSize">
          <size>
           <width>662</width>
           <height>80</height>
          </size>
         </property>
         <widget class="QLabel" name="lb_title">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>20</y>
            <width>291</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>SimSun-ExtB</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设备管理系统</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="right_wgt" native="true">
         <property name="minimumSize">
          <size>
           <width>251</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>251</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QPushButton" name="pb_resize">
          <property name="geometry">
           <rect>
            <x>185</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>最大化</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image:url(:/MyFramework/images/放大-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
    background-image: url(:/MyFramework/images/放大-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/MyFramework/images/放大-默认.png);
    border:0px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_min">
          <property name="geometry">
           <rect>
            <x>155</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>最小化</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image:url(:/MyFramework/images/最小化-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
    background-image: url(:/MyFramework/images/最小化-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/MyFramework/images/最小化-默认.png);
    border:0px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lb_user">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>25</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(15, 11, 49);</string>
          </property>
          <property name="text">
           <string>admin</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_close">
          <property name="geometry">
           <rect>
            <x>215</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>关闭</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image: url(:/MyFramework/images/关闭-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
	background-image: url(:/MyFramework/images/关闭-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/MyFramework/images/关闭-默认.png);
    border:0px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_min_1">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>当前用户</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color:transparent;
	background-image: url(:/MyFramework/images/用户1.png);
    border:0px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="center_wgt" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="top_wgt_2" native="true">
      <property name="minimumSize">
       <size>
        <width>1014</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> 
background-color: rgb(85, 170, 255);</string>
      </property>
      <widget class="QWidget" name="wgt_status_left" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>790</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>790</width>
         <height>24</height>
        </size>
       </property>
       <widget class="QLabel" name="lb_flag_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>4</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="MyFramework.qrc">:/MyFramework/images/icon_info.png</pixmap>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyFramework.qrc"/>
 </resources>
 <connections/>
</ui>
